.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+## Asscociated Issue
+ 
+Replace this text with the Jira issue resolved (e.g. CASSANDRA-####) or text specifying why this is a trivial patch.
+
+## Author(s) and Reviewers
+
+Patch by &lt;author>; reviewed by <reviewer[s]>
+
+Co-authored-by: First Last <author1@domain.com><br/>
+Co-authored-by: First2 Last2 <author2@domain.com>
+
+
+## What the patch does
+
+Replace this text with a detailed commit paragraph of why and how the change was made and relevant context.
+
+##  Recommendations:
+
+if not a trivial change:
+  - Jira ticket contains a description of: what is fixed, why it is needed, and what branches to apply it to.
+  - Tests are included.
+  - Documentation changes and/or updates are included.
+  
+
+## Notice
+By submitting this pull request, I acknowledge that I am making a contribution to the Apache Software Foundation under the terms and conditions of the Contributor's Agreement.
+
+###References:
+* The [Apache Cassandra "Contributing to Cassandra" guide](https://cassandra.apache.org/_/development/index.html) 
+* The [Apache Cassandra "Working on Documentation" guide](https://cassandra.apache.org/_/development/documentation.html)
+* The [Apache Contributor's Agreement](https://www.apache.org/licenses/contributor-agreements.html).
```

Comment:
i say remove this. it is not required and will only scare people off.

Reply:
I disagree about this scaring people off. The second paragraph in the referenced page is:

> These agreements help us achieve our goal of providing reliable and long-lived software products through collaborative, open-source software development. In all cases, contributors retain full rights to use their original contributions for any other purpose outside of Apache, while providing the ASF and its projects the right to distribute and build upon their work.

Which seems completely inclusive and reasonable to me.

Reply:
it's out of scope for the PR. we won't actually be asking the contributor for an ICLA. creates unnecessary confusion.

speaking from experience, i don't contribute simple one-liners PRs to other projects if i'm confronted with such a barrage of information. but that's just me.